src/main/java/com/market/servicemarket/util/GlobalExceptionHandler.java

```diff
@@ -12,49 +12,91 @@
 import org.springframework.web.context.request.WebRequest;
 import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
+import javax.mail.MessagingException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
 
 @ControllerAdvice
 public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
 
 
+    private JavaMailUtil mailUtil;
+
     @Autowired
     ConfigurationUtil configurationUtil;
 
+    String emails = "";
+    StringWriter errors = new StringWriter();
+
     @Override
     protected ResponseEntity<Object> handleHttpMessageNotReadable(HttpMessageNotReadableException ex, HttpHeaders headers, HttpStatus status, WebRequest request) {
+        emails = configurationUtil.getMessage(Constants.ADMIN_EMAILS);
         BaseResponse response = new BaseResponse();
         response.setResponseCode(Constants.INVALID_FIELD_RESPONSE_CODE);
         response.setResponseMessage(configurationUtil.getMessage(Constants.INVALID_FIELD_RESPONSE_CODE));
         ex.printStackTrace();
```

Comment:
Please remove printStackTrace, As error should not come on console now, For any error, admins will receive email